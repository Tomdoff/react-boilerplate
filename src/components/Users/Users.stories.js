import React from 'react';
import { storiesOf } from '@storybook/react';
import Users from './Users';
storiesOf('Users', module).add('Users', () => (
    <div>
        <Users />
    </div>
));
