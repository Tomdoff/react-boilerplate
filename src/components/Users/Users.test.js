import React from 'react';
import { shallow } from 'enzyme';
import Users from './Users';

describe('<Users />', () => {
    test('renders', () => {
        const wrapper = shallow(<Users />);
        expect(wrapper).toMatchSnapshot();
    });
});
