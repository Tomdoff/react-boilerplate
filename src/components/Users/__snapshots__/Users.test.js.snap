// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Users /> renders 1`] = `ShallowWrapper {}`;
